modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/document/library/DocumentLibraryDDMFormFieldTemplateContextContributorTest.java

```diff
@@ -153,6 +153,18 @@ public void testGetParametersShouldContainGuestUploadURL() {
 			CoreMatchers.containsString("param_folderId=" + _FORMS_FOLDER_ID));
 	}
 
+	@Test
+	public void testGetParametersShouldContainItemSelectorURL() {
+		DocumentLibraryDDMFormFieldTemplateContextContributor spy = createSpy(
+			mockThemeDisplay());
+
+		Map<String, Object> parameters = spy.getParameters(
+			new DDMFormField("field", "document_library"),
+			createDDMFormFieldRenderingContext());
+
+		Assert.assertTrue(parameters.containsKey("itemSelectorURL"));
```

Comment:
@carolmariaabb, can we also check if the `folderId` is present on the `itemSelectorURL` too?

Reply:
@natocesarrego good idea! I will add this and resend it here.